### ISSUE TITLE
Crash after a while of Paper trading

### ISSUE THREAD

**Issue by haihosilver** [askmike/gekko#1582]
```
  <-- POST /api/getCandles
  --> POST /api/getCandles 200 548ms 3.65kb
  <-- POST /api/getCandles
  --> POST /api/getCandles 200 549ms 3.65kb
  <-- POST /api/getCandles
  --> POST /api/getCandles 200 552ms 3.65kb
  <-- POST /api/getCandles
  --> POST /api/getCandles 200 550ms 3.65kb
/home/rolf/gekko/node_modules/ws/lib/WebSocket.js:219
    else throw new Error('not opened');
         ^

Error: not opened
    at WebSocket.send (/home/rolf/gekko/node_modules/ws/lib/WebSocket.js:219:16)
    at _.each.client (/home/rolf/gekko/web/server.js:28:22)
    at Function.forEach (/home/rolf/gekko/node_modules/lodash/dist/lodash.js:3298:15)
    at broadcast (/home/rolf/gekko/web/server.js:26:5)
    at pipelineRunner (/home/rolf/gekko/web/routes/startGekko.js:141:5)
    at Object.message (/home/rolf/gekko/core/workers/pipeline/messageHandlers/realtimeHandler.js:15:9)
    at ChildProcess.<anonymous> (/home/rolf/gekko/core/workers/pipeline/parent.js:24:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:772:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
disconnect
disconnect
disconnect
disconnect
disconnect
disconnect
```

**Comment by askmike** [comment]
Will be fixed asap, the problem has to do with your browser having an unreliable connection to the server. There is a fix (c7aedf2bd424279e4984a13895472f187263a286) in another branch (#1850) that will be merged soon (with additional features).

**Comment by askmike** [comment]
I am closing this for now:

- Will be fixed in the next release.
- If you run into this, be sure to use chrome or firefox and be sure to stay well connected to the gekko server! If you can't get around to this for now maybe fall back to the commandline version.
